### DISCUSSION TITLE
Is it possible to get TPM state using iPXE commands/settings?

### DISCUSSION THREAD

**Discussion by mr-const** [ipxe/ipxe#1010] [Q&A]
When booting a windows image I need to provision it differently based on presence and activity of a TPM module. Is it possible to get this information about a TPM state using present tools, or some coding work has to be done?

**Comment by NiKiZe** [comment]
TPM presence could be available thru smbios tables, if it is you should be able to see it in `dmidecode` and from there find the details. (I might be wrong, but I scrolled by something about this in the SMBIOS spec the other day)
